### ISSUE TITLE
Update dependencies

### ISSUE THREAD

**Issue by DywiTom** [valor-software/ng2-charts#1350]
Please update dependencies,.
Thank you!

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: project@1.0.0
npm ERR! Found: rxjs@7.4.0
npm ERR! node_modules/rxjs
npm ERR!   rxjs@"^7.4.0" from the root project
npm ERR!   peer rxjs@"^6.5.3 || ^7.4.0" from @angular/common@13.0.3
npm ERR!   node_modules/@angular/common
npm ERR!     @angular/common@"~13.0.3" from the root project
npm ERR!     peer @angular/common@">=11.0.0" from ng2-charts@3.0.0
npm ERR!     node_modules/ng2-charts
npm ERR!       ng2-charts@"^3.0.0" from the root project
npm ERR!   1 more (@angular/core)
```

**Comment by santam85** [comment]
Released v3.0.1 with a fixed peerDependency range. Could you validate it now?

**Comment by DywiTom** [comment]
It works now, thanks!
